alws/build_planner.py

```diff
@@ -91,6 +91,16 @@ def load_platform_flavors(self, flavors):
         if db_flavors:
             self._platform_flavors = db_flavors
 
+    def is_beta_build(self):
+        if not self._platform_flavors:
```

Comment:
`return bool(self._platform_flavors)`

Reply:
I cannot just return at this point because it should return only when no flavors are there